matching-notification-api/src/main/java/org/phenotips/matchingnotification/notification/internal/DefaultUserPatientMatchEmail.java

```diff
@@ -142,24 +146,31 @@ protected void setFrom()
     {
         super.setFrom();
         try {
-            Address userEmails = getUserEmails();
+            Set<Address> userEmails = getUserEmails();
             if (userEmails != null) {
```

Comment:
`getUserEmails()` is a private method that always returns a non-null result. There's no need to keep this null check.